### DISCUSSION TITLE
Which pi fits best?

### DISCUSSION THREAD

**Discussion by lenlennart** [MiczFlor/RPi-Jukebox-RFID#1487] [Q&A] [answered]
Hello,

First of: great project! I have been thinking about buying a toniebox for some time, but now I wanna try out a phoniebox first.

Which pi (zero, 3, 3+ or 4) fits the purpose best?
Is there any delay in processing of a RFID chip or in playback (from Spotify) when using a pi zero? Does the extra power of the 3/3+/4 surve any purpose?
Else I assume pi zero is the best option for it's low power consumption (when used with a powerbank)? Only downside send to be the extra USB board required.

Thanks
Lennart

**Comment by pabera** [comment, marked as answer]
I have a 3+ and it works fine. I believe a 4 is an overkill.
1, 2 and Zero are similar and probably work as well, many members of this forum have tried them out successfully. Maybe you can do a search. It depends all a bit on what you wanna do, like the box form factor and the type of accessories you want to add. Me, for instances, I have less experience with amplifiers, soldering etc so I needed the USB connectors of the 3+ to add all the required devices like USB powered speakers, RFID reader etc. I was following this [INSTALL Guide](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/docs/INSTALL-COMPLETE-GUIDE.md).

**Reply by s-martin** [reply]
With the Zero many people experience performance issues, particularly boot time and when using Spotify.
Please search the issues and discussions here, some issues can be solved.

If you don't need the small form factor I would go for a 3(+).

**Reply by lenlennart** [reply]
Ohh thanks for the info! I was about to order the zero, but 3min boot time sounds really troublesome... pi3 it is then!

**Reply by MiczFlor** [reply]
Hi @lenlennart 
there is a "new" version coming up, but possibly not in time for your Phoniebox. The version 3 is a re-write of the entire code base. And it is not stable yet for production. And it was written in mind also to better support the pi zero. But for you, the 3+ and the latest version 2.x will be the best choice.

**Comment by lenlennart** [comment]
Has anyone tried the new Zero 2 W with the phoniebox? Very keen about boottime.

**Reply by passuff** [reply]
I have a fresh installation (spotify) on Zero W2 and from pushing the button to the startup sound it takes 53s.

**Reply by lenlennart** [reply]
ok, thanks! This is pretty much the same as my pi 3b.
